pipelines/coref/scripts/create_eval_data.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
```

Comment:
Is this script still being used? I don't see it referenced from the yml file? Similarly, do we still need this line in the yml?
```
rm -f corpus/test.key corpus/test.response
```

Reply:
You're right, I think that line and this script are no longer necessary. I'm kind of tempted to leave the script around in case someone wanted to run the official eval scripts on their own, but I'll remove it for now.